poetry_scripts.py

```diff
@@ -66,8 +66,8 @@ def generate_docs():
     check_exit_code(
         "pdoc \
     --include-undocumented \
-    --favicon https://logo.swmansion.com/membrane/\?width\=100\&variant\=signetDark\
-    --logo https://logo.swmansion.com/membrane/\?width\=70\&variant\=signetDark\
+    --favicon https://logo.swmansion.com/membrane/?width=100&variant=signetDark\
+    --logo https://logo.swmansion.com/membrane/?width=70&variant=signetDark\
```

Comment:
Are you sure un-escaping the ampersand `&` here doesn't cause the command to break?

Reply:
Ah sorry for those changes I forgot to untrack this file.

If you are using python3 all of those escaping is unnecessary and throws an error.